src/components/editor/editor.tsx

```diff
@@ -60,7 +61,12 @@ export const Editor: React.FC = () => {
 
   useEffect(() => {
     setDocumentContent(editorTestContent)
-  }, [])
+    const requestedMode = search.substr(1)
+    const mode = Object.values(EditorMode).filter(mode => mode === requestedMode)
```

Comment:
Use find instead of filter

Reply:
Good point, I just forgot about that method. Will change that soon.

Reply:
Done